src/main/java/pages/LaunchPage.java

```diff
@@ -10,39 +10,87 @@
     private static final String GET_STARTED_ID = "Get started";
     private static final String LAUNCH_PAGE_IMAGE = "XCUIElementTypeImage";
     private static final String BETA_TEXT = "BETA";
-    private static final String TITLE_1= "Vehicle Standards";
+    private static final String TITLE_1 = "Vehicle Standards";
     private static final String TITLE_2 = "Assessment";
     private static final String SUBTITLE = "Mobile Test Capture";
 
+
     public void clickGetStarted() {
+
         if (!TestHandler.getInitializedStatus().get()) {
-            longWaitUntilPageIsLoadedByIdAndClickable(GET_STARTED_ID).click();
+            longWaitUntilPageIsLoadedByIdAndClickable(GET_STARTED_ID);
+            clickToEnableOrDisable();
+
+            findElementById(GET_STARTED_ID).click();
             TestHandler.getInitializedStatus().set(true);
 
         } else {
-//            ((IOSDriver)((WebDriverFacade) getDriver()).getProxiedDriver()).closeApp();
-//            ((IOSDriver)((WebDriverFacade) getDriver()).getProxiedDriver()).launchApp();
-            shortWaitUntilPageIsLoadedByIdAndClickable(GET_STARTED_ID).click();
+
+            ((IOSDriver) ((WebDriverFacade) getDriver()).getProxiedDriver()).closeApp();
+            ((IOSDriver) ((WebDriverFacade) getDriver()).getProxiedDriver()).launchApp();
+            shortWaitUntilPageIsLoadedByIdAndClickable(GET_STARTED_ID);
+            clickToEnableOrDisable();
+            findElementById(GET_STARTED_ID).click();
         }
 
     }
 
+
+    public void clickToEnableOrDisable() {
+        try {
+            Thread.sleep(700);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        if (TestHandler.getIsEnableStatus().get()) {
+
+            int initial = 0;
+            while (initial != 5) {
+
+
+                try {
+                    Thread.sleep(300);
+                    initial++;
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+
+                tapByCoordinates(70, 140);
```

Comment:
why not tap on "//XCUIElementTypeOther[@name="Group"]/XCUIElementTypeImage" instead?